setup.py

```diff
@@ -131,7 +131,7 @@
     "aiohttp",
     # To get datasets from the Datasets Hub on huggingface.co
     # minimum 0.14.0 to support HfFileSystem
-    "huggingface-hub>=0.14.0,<1.0.0",
+    "huggingface_hub>=0.18.0rc0",
```

Comment:
to apply once hfh 0.18 is released
```suggestion
    "huggingface_hub>=0.18.0",
```